frontend/src/features/data_entry/components/DataEntrySection.tsx

```diff
@@ -59,40 +59,34 @@ export function DataEntrySection({ sectionId }: { sectionId: FormSectionId }) {
     }
   }
 
-  const [missingTotalError, setMissingTotalError] = React.useState(false);
-
   // Memoize getCodes() results to prevent unnecessary focus triggers in Feedback
-  const memoizedErrorCodes = React.useMemo(() => formSection.errors.getCodes(), [formSection.errors]);
+  const memoizedErrorCodes = React.useMemo(() => {
+    const errorCodes = formSection.errors.getCodes();
+    const missingTotalErrorIndex = errorCodes.indexOf("F402", 0);
+    if (missingTotalErrorIndex > -1) {
+      errorCodes.splice(missingTotalErrorIndex, 1);
+    }
+    return errorCodes;
+  }, [formSection.errors]);
   const memoizedWarningCodes = React.useMemo(() => formSection.warnings.getCodes(), [formSection.warnings]);
+  const onlyMissingTotalError = formSection.errors.includes("F402") && formSection.errors.size() === 1;
 
   React.useEffect(() => {
-    if (missingTotalError && totalFieldId) {
+    if (formSection.errors.includes("F402") && totalFieldId) {
       document.getElementById(totalFieldId)?.focus();
     }
-  }, [missingTotalError, totalFieldId]);
+  }, [formSection.errors, totalFieldId]);
```

Comment:
I think it makes sense to move this down to `InputGrid` or `InputGridRow` (@oliver3 already suggested something similar to me)